src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -184,6 +190,11 @@ class Alpha_complex {
  private:
   template<typename InputPointRange >
   void init_from_range(const InputPointRange& points) {
+    #if CGAL_VERSION_NR < 1050000000
+    if (Is_Epick_D<Kernel>::value)
+      std::cerr << "It is strongly advised to use a CGAL version from 5.0 for performance reasons." << std::endl;
```

Comment:
Er, the warning should be for people using CGAL-4.x with **Epeck_d**. For Epick_d, it doesn't matter if they have 4.x or 5.0.

Reply:
cf. ea296b4a93c37b0e19d4605c1be3e950ed8cd3c2